### PR TITLE
LanguageToolBear.py: Resolve settings conflict

### PR DESCRIPTION
**Pull request by pratyushprakash**
Rename setting `language` to `natural_language`
in LanguageToolBear.py to resolve conflict

Closes https://github.com/coala/coala-bears/issues/1137

### CODE REVIEW

File: bears/natural_language/LanguageToolBear.py [hunk 1]
```diff
@@ -33,16 +33,16 @@ def check_prerequisites(cls):
             except ImportError:  # pragma: no cover
                 return 'Please install the `language-check` pip package.'
 
-    @deprecate_settings(language='locale')
+    @deprecate_settings(natural_language='locale')
```
Comment (sils) [inline bears/natural_language/LanguageToolBear.py]: argh that means that if the user used `language` before this won't work and he won't get a deprecation warning, that's not ok. I'm not sure if the decorator is capable of two deprecations of the same variable but we gotta figure it out :/

Comment (pratyushprakash) [inline bears/natural_language/LanguageToolBear.py]: Oops. My apologies for forgetting about that.

Comment (sils) [inline bears/natural_language/LanguageToolBear.py]: I think we'll also want to have tests that verify that all deprecated variants work properly

**Comment by pratyushprakash** [comment]
Waiting for https://github.com/coala/coala/issues/3537 to be closed.

File: tests/natural_language/LanguageToolBearTest.py [hunk 2]
```diff
@@ -39,6 +39,20 @@
...
+    invalid_files=('Ein korrekter englischer Satz klingt für alle gut.',),
+    settings={'language': 'en-US'})
+
+
+LanguageToolBearDeprecatedSettingTest1 = verify_local_bear(
+    LanguageToolBear,
+    valid_files=('A correct English sentence sounds nice to everyone.',),
+    invalid_files=('Ein korrekter englischer Satz klingt für alle gut.',),
+    settings={'locale': 'en-US'})
+
```
Comment (Asnelchristian) [inline tests/natural_language/LanguageToolBearTest.py]: :+1: , thanks for adding these tests, we can be sure that the other deprecated settings work

File: tests/natural_language/LanguageToolBearTest.py [hunk 3]
```diff
@@ -39,6 +39,20 @@
...
 
 
+LanguageToolBearDeprecatedSettingTest1 = verify_local_bear(
+    LanguageToolBear,
+    valid_files=('A correct English sentence sounds nice to everyone.',),
+    invalid_files=('Ein korrekter englischer Satz klingt für alle gut.',),
+    settings={'language': 'en-US'})
+
+
+LanguageToolBearDeprecatedSettingTest1 = verify_local_bear(
```
Comment (Asnelchristian) [inline tests/natural_language/LanguageToolBearTest.py]: maybe ``LanguageToolBearDeprecatedSettingTest2`` instead

Comment (Asnelchristian) [inline tests/natural_language/LanguageToolBearTest.py]: so that we can easily identify which test is failing

Comment (pratyushprakash) [inline tests/natural_language/LanguageToolBearTest.py]: Oops that was by mistake. Meant to do that.

**Comment by Asnelchristian** [comment]
this looks good, to me. but let's wait for Ci

**Comment by Asnelchristian** [comment]
aouch..... three test are failing

**Comment by pratyushprakash** [comment]
@Asnelchristian Yeah, circle is not using the latest coala

File: bears/natural_language/LanguageToolBear.py [hunk 4]
```diff
@@ -33,16 +33,16 @@ def check_prerequisites(cls):
             except ImportError:  # pragma: no cover
                 return 'Please install the `language-check` pip package.'
 
-    @deprecate_settings(language='locale')
+    @deprecate_settings(natural_language=('language', 'locale'))
```
Comment (Asnelchristian) [inline bears/natural_language/LanguageToolBear.py]: i wonder, is multiple-deprecation allowed yet?

Comment (Asnelchristian) [inline bears/natural_language/LanguageToolBear.py]: uhm.... i don't think so :(

Comment (pratyushprakash) [inline bears/natural_language/LanguageToolBear.py]: I added this in https://github.com/coala/coala/pull/3546

**Comment by pratyushprakash** [comment]
The problem is circle is not using the newest coala, where multiple deprecations are supported.

**Comment by Asnelchristian** [comment]
ack https://github.com/coala/coala-bears/pull/1287/commits/735dda7583d6221d60b147f17b23685633d8e0f9

**Comment by Asnelchristian** [comment]
@rultor merge

**Comment by pratyushprakash** [comment]
Yayyyy. I'm gonna return the favor and review a few now.. :)

**Pull request merged**